cockroachcloud/cmek.md

```diff
@@ -108,6 +108,7 @@ Backups in {{ site.data.products.dedicated }} are triggered in two ways, only on
 ## Limitations
 The CMEK feature has the following limitations:
 
+- During limited access, CMEK is not available for {{ site.data.products.dedicated }} clusters on Azure. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
```

Comment:
That's good feedback. Yeah, it would be great to have a consistent positive note for all such sections, indicating that we are looking for feedback to plan GA and further phases.

Reply:
I've adjusted this a bit while trying to keep future-facing language to a minimum. PTAL.